server/stats/achievement_service.py

```diff
@@ -61,7 +62,7 @@
 
 
 @with_logger
-class AchievementService:
+class AchievementService(Service):
```

Comment:
Yea it doesn't have any effect now that I'm moving the auto-service creation to a different PR. But with those changes it will mean that the class is instantiated and set up automatically. For now it's kindof just a marker I guess.